ui/src/reusable_ui/components/wizard/WizardProgressBar.scss

```diff
@@ -0,0 +1,54 @@
+@import 'src/style/modules/influx-colors';
+@import 'src/style/modules/variables';
+
+.progress-bar {
```

Comment:
Same comment as previous about namespacing using `wizard` to avoid potential collisions